app/pages/search/controls/SearchControlsContainer.js

```diff
@@ -176,7 +176,7 @@ class UnconnectedSearchControlsContainer extends Component {
                 </Col>
               </Row>
               <Row>
-                <Col className="app-SearchControlsContainer__control" md={4} sm={6}>
+                <Col className="app-SearchControlsContainer__control" md={4} sm={12}>
```

Comment:
There is still 6 columns inside a single row, we should separate the 3 dropdowns to their own row element.